index.js

```diff
@@ -112,13 +141,33 @@ console.log(array1.fill(6));
     - Besides the methods on Person.prototype, babies have the ability to `.play()`:
         + Should return a string "Playing with x", x being the favorite toy.
 */
-function Baby() {
+const Baby=function (name,age,favoriteToy) {
   this.name=name;
   this.age=age;
+  this.favoriteToy=favoriteToy;
+  //this.play=function()[
+    //return 'Playing with x';
+  //]
+}
   
-
+this.play = function() {
+  return `Playing with ${this.favoriteToy}`
 }
 
```

Comment:
```js
this.play = function() {
  return `Playing with ${this.favoriteToy}`
}

This code was not required and doesn't do anything for you since its outside the code block `{}` of `function Baby()`